src/Controllers/DisplayApplicantsController.php

```diff
@@ -41,26 +41,8 @@ public function __invoke(Request $request, Response $response, array $args)
         $sortValue = $request->getQueryParams()['sort'] ?? '';
```

Comment:
why are you creating this variable to just assign it straight into an array?

Reply:
I didn't do this but good catch, i'll change it